src/main/java/org/milkteamc/autotreechop/AutoTreeChop.java

```diff
@@ -28,16 +41,56 @@ public class AutoTreeChop extends JavaPlugin implements Listener, CommandExecuto
 
     private Map<UUID, PlayerConfig> playerConfigs;
     private AutoTreeChopAPI api;
-    private String enabledMessage;
-    private String disabledMessage;
-    private String noPermissionMessage;
-    private String hitmaxusageMessage;
-    private String hitmaxblockMessage;
-    private String usageMessage;
-    private String blocksBrokenMessage;
-    private String enabledByOtherMessage;
-    private String disabledByOtherMessage;
-    private String consoleName;
+
+    public static final Message ENABLED_MESSAGE = new MessageBuilder("enabled")
+            .withDefault("<positive>Auto tree chopping enabled.</positive>")
+            .withTranslation(Locale.GERMAN, "<positive>Auto-Baumf�llen aktiviert.</positive>")
```

Comment:
I don't think this is a good idea to write translated messages in code

Reply:
Fair enough. So would you want me to keep english but move german to a resource that will be saved to file on first launch?